eng/common/scripts/Prepare-Release.ps1

```diff
@@ -177,8 +177,17 @@ if ($releaseTrackingOnly)
 
 if (Test-Path "Function:SetPackageVersion")
 {
-  SetPackageVersion -PackageName $packageProperties.Name -Version $newVersion -ServiceDirectory $packageProperties.ServiceDirectory -ReleaseDate $releaseDateString `
-    -PackageProperties $packageProperties
+  $replaceLatestEntryTitle = $true
+  $latestVersion = Get-LatestReleaseDateFromChangeLog -ChangeLogLocation $packageProperties.ChangeLogPath
+  if ($latestVersion -ne $null)
```

Comment:
```suggestion
  if ($latestVersion)
```

tip you don't need to explicitly test for null or empty the value will result in the correct true/false value based on that.